### ISSUE TITLE
setCurrentRecording to binary file in multiFileMode returns that selected recordings were not found in recording list.

### ISSUE THREAD

**Issue by alinaheukamp** [neural-electrophysiology-tool-team/NET#8]
In MultiFileMode, setCurrentRecording to the name of the binary file returns the message
"Selected Rcording/s  were not found in the recording list"

The binary file name now consists of all recNames concatenated and joined by '-'.

In recAnalysis, line 341 and onwards (setCurrentRecording):
- selectedSubValues is a 1x1 cell array that contains all renames joined by '-' - does this need to be split again to be a cell array of dimensions number-of-filesx1? -- strsplit(selectedSubValues,'-')?
- currently nRec is 0

**Comment by alinaheukamp** [comment]
I think I solved it, but I didn't commit it yet. I created another issue today (19/2/2020) and would like to commit my change to this issue here only after the other one is solved to see that my new solution still works.

**Comment by yuvalorsher** [comment]
This isn't a code problem but an excel sheet problem. 

The new "combined" name needs to be appended to the excel sheet and then MEAAnalysis re-initiated with the added line. 

Alternatively, we can append the combined bin to the object within MEAAnalysis which represents the excel table and then offer to save the updated excel.

**Comment by yuvalorsher** [comment]
I fixed the problem multi-file but it will change the workflow a little bit (as follows):

Instead of: 
```{Matlab}
MA.setCurrentRecording(strcat('recNames=',recname1,'.bin,recFormat=binaryRecording'));
```
You would write:
```{Matlab}
MEA.addBinary2Table('updateExternalExcel',1,'setCurrent2Bin',1);
```
_This line can then be followed by your normal workflow (e.g., sorting). _

This will add the binary file to your excel sheet (internally as well as externally). Make sure it's not locked (e.g., open elsewhere). And it will set the current data object to the produced bin file. 

Since we won't need bin entries for every h5 we produce using the convertuploaddocument.py anymore - it might be worthwhile to modify this code so it doesn't produce "bin" lines.  If you do that on your branch and then initiate a pull-request we'll review it and integrate it. 

If you still have any trouble, feel free to re-open this issue.
